pkg/extensions/tbtc/recovery/storage.go

```diff
@@ -147,11 +149,28 @@ func (dis *DerivationIndexStorage) GetNextIndex(extendedPublicKey string) (uint3
 		return 0, err
 	}
 
-	index, err := dis.read(extendedPublicKey)
+	lastIndex, err := dis.read(extendedPublicKey)
 	if err != nil {
 		return 0, err
 	}
-	return uint32(index + 1), nil
+	nextIndex := uint32(lastIndex + 1)
+	numberOfTries := uint32(100)
```

Comment:
Thoughts around having this property configurable?

Reply:
I'm down - but I do want to check before I write that up that number of tries is the retry strategy we want to figuring out how long to spend on finding an unused recovery address